### PR TITLE
Add Team-based FixVersion report

### PR DESCRIPTION
**Pull request by josh-mckenzie**
This also tidies up the formatting for team reports by adding some flags to the
DisplayFilter scope, namely to suppress dependency resolution regardless of
global settings, and also allowing a report to select between the pager and a
local print.

### CODE REVIEW

**Comment by josh-mckenzie** [comment]
@jillichrome or @djbowers - either of you have bandwidth for a review?

File: src/member_issues_by_status.py [hunk 1]
```diff
@@ -73,10 +72,10 @@ def __init__(self, jira_user_name):
         self._aliased_names = {}
 
         # WARNING: These containers also need to be reflected in self.clear and self.sort_tickets
-        self.assigned = []
-        self.closed = []
-        self.reviewer = []
-        self.reviewed = []
+        self.assigned = []  # type: List[JiraIssue]
```
Comment (djbowers) [inline src/member_issues_by_status.py]: You can write your type annotations in python 3 like this:
`self.assigned: List[JiraIssue] = []`

Comment (josh-mckenzie) [reply src/member_issues_by_status.py]: Oh. That's much cleaner. I'll do that then push and merge. Thanks for the heads up - was irritated about that when I wrote it (the inconsistency w/Python 3's PEP 484 impl, or at least what I could see of it)

**Comment by josh-mckenzie** [comment]
Looks like your syntax is slightly off @djbowers, should read:
`self.assigned = List[JiraIssue]`

Cleaned up some annotations, checked static offline, tested, and force pushed to tidy up a commit that was incomplete. Looks like it blasted your comment, but waiting on travis to be happy before merge.

**Comment by josh-mckenzie** [comment]
And now that I'm in 3.6, syntax is updated, and changed check_codebase.sh to take an argument to specify a python command (for those of us not in virtualenv)

**Comment by josh-mckenzie** [comment]
@djbowers @MichaelHamm Could one of you do a quick once-over on the changes to check_codebase.sh to make sure I didn't do something blindingly stupid? Will merge after.

**Comment by djbowers** [comment]
Looks good to me. I ran both 
`$ source check_codebase.sh python`
and
`$ source check_codebase.sh`
and both worked fine. However the latter did spit this out 
`-bash: [: =: unary operator expected`

Side note: would it be possible for this script to check for pip dependencies flake8, pycodestyle, and pylint, and install them if they aren't already? That would be cool.

**Comment by josh-mckenzie** [comment]
Hm. I always just did `./check_codebase.sh` to let it run, since the interpreter designation at the top should let it know it's a bash script.

I like the idea of the dep install. Create an issue for it?

**Pull request merged**